packages/transitive-overlay/src/index.tsx

```diff
@@ -108,29 +120,24 @@ const TransitiveCanvasOverlay = ({
   transitiveData
 }: Props): JSX.Element => {
   const { current: map } = useMap();
-  useEffect(() => {
-    if (!map) return;
```

Comment:
This line was a bug, I believe, since the map is checked for again below in the following if statement.